understanding/orientation.html

```diff
@@ -58,155 +56,112 @@
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.3.4:</span>Orientation (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.3.4:</span>表示の向き (レベル AA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Content does not restrict its view and operation to a single display orientation,
-                     such as portrait or landscape, unless a specific display orientation is <a href="#dfn-essential">essential</a>.
+                  <p>コンテンツは、その表示及び操作を、縦向き (portrait) 又は横向き (landscape) のような単一のディスプレイの向きに制限しない。ただし、特定のディスプレイの向きが<a href="#dfn-essential">必要不可欠</a>な場合は除く。
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Examples where a particular display orientation may be essential are a bank check,
-                        a piano application, slides for a projector or television, or virtual reality content
-                        where content is not necessarily restricted to landscape or portrait display orientation.
+                     <p class="note-title marker">注記</p>
+                     <p>特定のディスプレイの向きが必要不可欠となり得る例としては、銀行の小切手、ピアノのアプリケーション、プロジェクターもしくはテレビ向けのスライド、又はコンテンツが必ずしも横向き (landscape) もしくは縦向き (portrait) のディスプレイの向きに制限されないバーチャルリアリティのコンテンツが挙げられる。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   				
-                  <dt>Goal</dt>
-                  <dd>Devices can be used in any orientation.</dd>
+                  <dt>目標</dt>
+                  <dd>デバイスがどの向きであっても使用できるようにする。</dd>
                   				
-                  <dt>What to do</dt>
-                  <dd>Don't lock content to either portrait or landscape presentation.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コンテンツを、縦向き (portrait) 又は横向き (landscape) のいずれかの提示に固定しない。</dd>
                   				
-                  <dt>Why it's important</dt>
-                  <dd>Wheelchair users and others may have devices mounted in a fixed orientation.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>車いすの利用者など、固定された向きでデバイスが取り付けられている可能性がある。</dd>
```

Comment:
「椅子」は常用漢字表にあるので（解説書本文がそうなっているように）漢字になりますね…
```suggestion
                  <dd>車椅子の利用者など、固定された向きでデバイスが取り付けられている可能性がある。</dd>
```